### DISCUSSION TITLE
Multiple languages streaming in live node starter

### DISCUSSION THREAD

**Discussion by alexopoulos7** [deepgram/community#891] [General help] [answered]
Hello, I am trying to change the [deepgram-live-node-starter](https://github.com/deepgram-starters/live-node-starter) so that it supports multiple languages. I have [forked here](https://github.com/alexopoulos7/live-node-starter-multiple-languages) the code and added multiple languages in server but also in UI and client code.
When you click start it works only for English and not for the other languages.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@alexopoulos7 When you say 

> so that it supports multiple languages.

What specifically are you trying to do with multiple languages?

**Comment by alexopoulos7** [comment]
I want users to be able to select the language of  discussion from the
dropdown menu
Angelos

**Comment by jpvajda** [comment, marked as answer]
@alexopoulos7  I tested your code out, it seems the language switching works, but the UI isn't updating as I switch between the languages.  I think you might want to look at how you are handling state in your UI and ensure you are actually passing back the response from Deepgram correctly to the UI as a user switches the language option in the drop down.
